plugins/auth-backend/api-report.md

```diff
@@ -47,6 +47,14 @@ export type AtlassianProviderOptions = {
   };
 };
 
+// @public (undocumented)
```

Comment:
might as well slap the docs on these while we're at it? :)

Reply:
Eh, none of them have it so would rather deal with all of them at once at some other point x)